I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 57, in script
    $ initializing_things()
  File "game/script.rpy", line 57, in <module>
    $ initializing_things()
      ~~~~~~~~~~~~~~~~~~~^^
  File "game/initializing_things.rpy", line 1298, in initializing_things
    renpy.say(mc, equipment.equipment_id)
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: unsupported operand type(s) for %: 'int' and 'TagQuotingDict'

-- Full Traceback ------------------------------------------------------------

Traceback (most recent call last):
  File "game/script.rpy", line 57, in script
    $ initializing_things()
  File "renpy/ast.py", line 1187, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/python.py", line 1260, in py_exec_bytecode
    exec(bytecode, globals, locals)
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "game/script.rpy", line 57, in <module>
    $ initializing_things()
      ~~~~~~~~~~~~~~~~~~~^^
  File "game/initializing_things.rpy", line 1298, in initializing_things
    renpy.say(mc, equipment.equipment_id)
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/exports/sayexports.py", line 118, in say
    what = what % tag_quoting_dict
           ~~~~~^~~~~~~~~~~~~~~~~~
TypeError: unsupported operand type(s) for %: 'int' and 'TagQuotingDict'

Windows-11-10.0.26120-SP0 AMD64
Ren'Py 8.4.1.25072401
Protectors 1.0
Mon Aug 11 16:52:21 2025
